index.html

```diff
@@ -1,278 +1,375 @@
+        </div>
+      </div>
+
+      <footer class="desktop-v">
+        <section>
+          <img src="assets/icons/image_geometry_1.svg" alt="" />
+          <h4 class="get-started">Get started</h4>
+          <p class="get-paragraph get-para">
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </p>
+        </section>
+        <section class="form-container">
+          <form action="https://formspree.io/f/xwkjrzvw" method="post">
+            <ul class="section-form">
+              <li>
+                <label class="label-1">your name</label>
+                <input
+                  class="name-text"
+                  type="text"
+                  id="name"
+                  name="user-email"
+                  placeholder="Thierry..."
+                  required
+                  maxlength="30"
+                />
+              </li>
+              <li>
+                <label class="label-1">your email</label>
+                <input
+                  class="email-border"
+                  type="text"
+                  id="email"
+                  name="email"
+                  placeholder="noumbuht@"
+                  required
+                />
+              </li>
+              <li>
+                <label class="label-1">your Message</label>
+                <textarea
+                  class="textarea1 textarea2"
+                  id="message"
+                  placeholder="Enter your message here..."
+                  required
+                  maxlength="500"
+                ></textarea>
+              </li>
+            </ul>
+            <div class="start-container">
+              <button type="button" class="Start-collaboration">
+                <p>Start collaboration</p>
+              </button>
+            </div>
+          </form>
+        </section>
+
+        <section class="rectangle_23">
+          <div class="rectangle_24">
+            <a class="get-resume" href="#"
+              ><span class="get">Get my Resume</span></a
+            >
+            <img src="assets/footer-img/ic_linkteal.svg" alt="download image" />
+          </div>
+          <ul class="social">
+            <li>
+              <img
+                src="assets/footer-img/ic_stackoverflow.svg"
+                alt="stackoverflow"
+              />
+            </li>
+            <li>
+              <img src="assets/footer-img/github.svg" alt="github logo" />
+            </li>
+            <li>
+              <img src="assets/footer-img/gtwitter.svg" alt="twitter logo" />
+            </li>
+            <li><img src="assets/footer-img/linkdin.svg" alt="linkdin" /></li>
+            <li>
+              <img src="assets/footer-img/medium.svg" alt="mediume logo" />
+            </li>
+            <li>
+              <img src="assets/footer-img/angellist.svg" alt="angellist logo" />
+            </li>
+          </ul>
+        </section>
```

Comment:
> - [ ] I noticed that your footer social icons on the desktop screen do not match the Figma design as much as possible. It would be highly appreciated if you fix that.👍 This would ensure a better user experience.
> 
> Figma design:
> 
> ![7-1](https://github.com/axcell-0/Thierrys-portfolio/assets/40009354/3b9cd219-d4c0-41f2-a5db-18d97efcc40e)
> 
> Your design:
> 
> ![7](https://github.com/axcell-0/Thierrys-portfolio/assets/40009354/d187d2ee-87b7-41bb-befc-9adb78622d95)
>